lib/master.js

```diff
@@ -195,9 +195,35 @@ Master.prototype._startWorkers = function(remainingWorkers) {
                 type: 'config',
                 body: yaml.dump(self.config)
             });
+
+            var workerExit = function(code) {
```

Comment:
@d00rman Then we need to move registration of the `cluster.on('exit'...` after all workers were started. In that case, if a worker dies after it was started, but before the last worker finished startup, we would loose a worker.